config/environments/development.rb

```diff
@@ -1,6 +1,15 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = true
+    Bullet.alert         = true
+    Bullet.bullet_logger = true
+    Bullet.console       = true
+    Bullet.rails_logger  = true
+    Bullet.add_footer    = true
+  end
+
```

Comment:
- [ ] Remember that we are not using the gem bullet, so please 🙏🏽 remove this piece of code to be able to create the database for running the integration specs for views.